Lib/multiprocessing/process.py

```diff
@@ -304,12 +304,12 @@ def _bootstrap(self):
             finally:
                 util._exit_function()
         except SystemExit as e:
-            if not e.args:
-                exitcode = 1
-            elif isinstance(e.args[0], int):
-                exitcode = e.args[0]
+            if e.code is None:
+                exitcode = 0
+            elif isinstance(e.code, int):
+                exitcode = e.code
             else:
-                sys.stderr.write(str(e.args[0]) + '\n')
+                sys.stderr.write(str(e.code) + '\n')
                 exitcode = 1
```

Comment:
Thinking  this exitcode. Why we need return a '1'?

Reply:
It aligns with the description of [`sys.exit`](https://docs.python.org/3/library/sys.html#sys.exit).

>  If another type of object is passed, `None` is equivalent to passing zero, and any other object is printed to `stderr` and results in an exit code of 1.

Reply:
Ok, thanks.